### PR TITLE
New landing testing as a service page

### PR DESCRIPTION
**Pull request by OleksandrDragun**

### CODE REVIEW

File: src/components/Heading/Heading.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const headingSize = {
+  h2: 'secondary-heading',
+  h3: 'third-heading',
+};
+
+export const Heading = ({ title, color, tag: Tag }) => (
+  <Tag className={classNames('heading', headingSize[Tag], color)}>{title}</Tag>
+);
+
+Heading.defaultProps = {
```
Comment (ViktorSoroka07) [inline src/components/Heading/Heading.jsx]: Let's not use default props but use default params for consistency

```
export const Heading = ({ title = 'h2', color = 'black-heading', tag: Tag }) => (

File: src/components/TestingAsServicePage/Tips/Tips.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Heading } from '../../Heading';
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/Tips/Tips.jsx]: Please check other places in PR
```suggestion
import React from 'react';

import { Heading } from '../../Heading';
```

File: src/components/TestingAsServicePage/Tips/Tips.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+import './Tips.scss';
+
+export const Tips = () => (
+  <div className="tips container">
+    <Heading title="Use Professional Service Hours across any service listed within" tag="h3" />
+    <div className="tips-lists">
+      <UnorderedList title="Manual testing" hasSeparator list={MANUAL_TESTING_LIST} />
+      <UnorderedList title="Automation" hasSeparator list={AUTOMATION_LIST} />
+      <UnorderedList title="Specialized" hasSeparator list={SPECIALIZED_LIST} />
+      <UnorderedList title="Exploratory" list={EXPLORATORY_LIST} />
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/Tips/Tips.jsx]: Please create array that you be mapped to render all items

`tipsList.map(tipsList => <UnorderedList key={tipsList.title} {...tipsList} />)`

File: src/pages/testing-as-a-service.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Layout } from '../components/Layout';
+import { TestingAsServicePage } from '../components/TestingAsServicePage';
+
+import '../styles/global.scss';
```
Comment (ViktorSoroka07) [inline src/pages/testing-as-a-service.jsx]: Please remove it here. It is not really needed. I think it was on the installation page when you were starting working on this page. I removed it already.

File: src/components/TestingAsServicePage/Challenges/Challenges.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      <Heading title="Challenges that Testing as a Service might help to solve" />
+      <div className="challenges__block">
+        <div className="challenges__block-item">Many defects from Production</div>
+        <div className="challenges__block-item">Automation debt exists on the project</div>
+        <div className="challenges__block-item">
+          Big regression testing phase happens periodically
+        </div>
+        <div className="challenges__block-item">No quick feedback from UAT phase</div>
+        <div className="challenges__block-item">Client request to cut costs</div>
+        <div className="challenges__block-item">Specialized testing required from time to time</div>
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/Challenges/Challenges.jsx]: Please map over array here

`challenges.map(challenge => <div key={challenge} className="challenges__block-item">{challenge}</div>)`

File: src/components/TestingAsServicePage/BenefitsOfTaas/icons/SpikeCapacity.jsx [hunk 6]
```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+
+export const SpikeCapacity = () => {
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/BenefitsOfTaas/icons/SpikeCapacity.jsx]: You don't need such components unless you want to style svg elements in CSS (e.g. on hover). Just store the file as SVG with `.inline.svg`  ending in name and then import as React component like this 
```
import SauceLabsIcon from './icons/saucelabs.inline.svg';
```

Please do the same for similar cases.

File: src/components/TestingAsServicePage/BenefitsOfTaas/BenefitsOfTaas.jsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+          </span>
+        </li>
+        <li>
+          <SpikeCapacity />
+          <span>Ability to spike capacity</span>
+        </li>
+        <li>
+          <HighStandard />
+          <span>Outcome-based model, with exceptionally high standards for quality</span>
+        </li>
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/BenefitsOfTaas/BenefitsOfTaas.jsx]: Please use array with interface bellow to map over this

`{ text, icon }[]`

File: src/components/TestingAsServicePage/Approach/ListDropdown/ListDropdown.scss [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 8px;
+    text-align: center;
+    cursor: pointer;
+    margin-bottom: 5px;
+
+    @include m.breakpoint(v.$tablet-sm-exact) {
+    }
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/Approach/ListDropdown/ListDropdown.scss]: Can be removed

File: src/components/TestingAsServicePage/Approach/Approach.scss [hunk 9]
```diff
@@ -0,0 +1,354 @@
...
+      &::before {
+        content: '';
+        background-color: var(--color-primary-300);
+        font-weight: bold;
+        position: absolute;
+        left: -20px;
+        width: 10px;
+        height: 10px;
+        border-radius: 50%;
+        top: 0;
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/Approach/Approach.scss]:
```suggestion
        position: absolute;
        content: '';
        top: 0;
        left: -20px;
        width: 10px;
        height: 10px;
        border-radius: 50%;
        background-color: var(--color-primary-300);
        font-weight: bold;
```

Not a big deal but please try to group related styles together (position related, display related (e.g. flex), box model related, font-related)

File: src/components/TestingAsServicePage/Approach/Approach.scss [hunk 10]
```diff
@@ -0,0 +1,354 @@
...
+      @include m.breakpoint(v.$tablet-sm-exact) {
+        &::before {
+          left: -54px;
+          top: 0;
+        }
+        &::after {
+          left: -50px;
+          top: 4px;
+        }
+      }
```
Comment (ViktorSoroka07) [inline src/components/TestingAsServicePage/Approach/Approach.scss]:
```suggestion
        &::before {
          left: -54px;
          top: 0;
        }

        &::after {
          left: -50px;
          top: 4px;
        }
      }
```

File: src/components/Heading/Heading.jsx (hunk 1 above)
Comment (OleksandrDragun) [reply src/components/Heading/Heading.jsx]: Done.

File: src/components/TestingAsServicePage/Tips/Tips.jsx (hunk 2 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/Tips/Tips.jsx]: Done.

File: src/pages/testing-as-a-service.jsx (hunk 4 above)
Comment (OleksandrDragun) [reply src/pages/testing-as-a-service.jsx]: Done.

File: src/components/TestingAsServicePage/Challenges/Challenges.jsx (hunk 5 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/Challenges/Challenges.jsx]: Done.

File: src/components/TestingAsServicePage/BenefitsOfTaas/icons/SpikeCapacity.jsx (hunk 6 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/BenefitsOfTaas/icons/SpikeCapacity.jsx]: Done. 
Thanks, I did not know this thing.

File: src/components/TestingAsServicePage/BenefitsOfTaas/BenefitsOfTaas.jsx (hunk 7 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/BenefitsOfTaas/BenefitsOfTaas.jsx]: Done.

File: src/components/TestingAsServicePage/Approach/Approach.scss (hunk 9 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/Approach/Approach.scss]: Done.
All styles was changed according to this order
https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style3.css

File: src/components/TestingAsServicePage/Approach/Approach.scss (hunk 10 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/Approach/Approach.scss]: Done.

File: src/components/TestingAsServicePage/Approach/ListDropdown/ListDropdown.scss (hunk 8 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/Approach/ListDropdown/ListDropdown.scss]: Done, thanks.

File: src/components/TestingAsServicePage/Tips/Tips.jsx (hunk 3 above)
Comment (OleksandrDragun) [reply src/components/TestingAsServicePage/Tips/Tips.jsx]: Done.

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**